muzzle/src/test/java/io/opentelemetry/javaagent/tooling/muzzle/VirtualFieldTestClasses.java

```diff
@@ -8,7 +8,7 @@
 import io.opentelemetry.context.Context;
 import io.opentelemetry.instrumentation.api.util.VirtualField;
 
-@SuppressWarnings({"ReturnValueIgnored", "unused"})
+@SuppressWarnings({"ReturnValueIgnored", "unused", "OtelPrivateConstructorForUtilityClass"})
```

Comment:
alternatively could add the constructor to the `Helper` class

Reply:
Fixed in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13600/commits/5ae38b4e9dda2a944ef8ac93d2837cccde997d65